lib/spring/client/run.rb

```diff
@@ -1,3 +1,4 @@
+require "spring/commands"
```

Comment:
@sonalkr132 @grosser I missed this, but we shouldn't be requiring `spring/commands` in the client. apart from this issue that @hsume2 reported, loading bundler in the client slows down the execution of spring commands, which is exactly what we want to avoid. The client needs to run as quickly as possible.

For client-side configuration, there is the `config/spring_client.rb` file, so in theory the user could set `Spring.quiet` in there (although I don't think this is really something we need to document...)

After fixing this, it would be great if we could somehow prevent a similar bug from being reintroduced in the future (perhaps we can somehow write a test which asserts that bundler is not present in the client? I don't know)